tests/Model/Client.php

```diff
@@ -6,6 +6,8 @@
 
 class Client extends User
 {
+    public $table = 'client';
```

Comment:
previously not needed, and now it is?

Reply:
Was discussed before but now I modified code to avoid changing this model